### ISSUE TITLE
Turn off github alerts until the cluster is reinstalled

### ISSUE THREAD

**Issue by 4n4nd** [operate-first/SRE#111]
Since we will be reinstalling the cnv/zero cluster soon, I think we should turn off the github alerts so that the repo does not get spammed with alert issues. We can turn them back on once the zero cluster is ready.

**Comment by HumairAK** [comment]
Yes let's turn them off so we don't get spammed as we bring it up, this will result in the github alert reciever being set to 0 replicas.
